Aplib.Core/Intent/Tactics/FirstOfTactic.cs

```diff
@@ -11,17 +11,25 @@ public class FirstOfTactic : AnyOfTactic
         /// <summary>
         /// Initializes a new instance of the <see cref="FirstOfTactic"/> class with the specified sub-tactics.
         /// </summary>
+        /// <param name="metadata">
+        /// Metadata about this tactic, used to quickly display the tactic in several contexts.
+        /// </param>
         /// <param name="subTactics">The list of sub-tactics.</param>
-        public FirstOfTactic(params Tactic[] subTactics) : base(subTactics)
+        public FirstOfTactic(Metadata metadata, params Tactic[] subTactics)
+            : base(metadata, subTactics)
         {
         }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="FirstOfTactic"/> class with the specified sub-tactics and guard condition.
         /// </summary>
-        /// <param name="subTactics">The list of sub-tactics.</param>
         /// <param name="guard">The guard condition.</param>
-        public FirstOfTactic(Func<bool> guard, params Tactic[] subTactics) : base(guard, subTactics)
+        /// <param name="metadata">
+        /// Metadata about this tactic, used to quickly display the tactic in several contexts.
+        /// </param>
+        /// <param name="subTactics">The list of sub-tactics.</param>
```

Comment:
```suggestion
        /// <param name="subTactics">The list of subtactics.</param>
```

Reply:
Could you actually ctrl+f sub-tactics and change it to subtactics everywhere in the solution?

Reply:
These lines are not currently touched by this PR. So I think if this needs to be fixed, dogmatically, it should be fixed in a different PR.

Reply:
Yeah I thought that might be the case